fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -0,0 +1,7 @@
+The official Zulip mobile app.
```

Comment:
I'd prefer that the Zulip app in F-Droid not be described as "official".

In particular, I see at https://gitlab.com/fdroid/fdroiddata/-/merge_requests/11516/diffs that there are a number of changes being made to the software, including removing some functionality. That's fine -- it's free software -- but it doesn't seem accurate to call the result "[t]he official Zulip" anything.